.github/workflows/ci.yml

```diff
@@ -39,16 +78,24 @@ jobs:
         name: 'Checkout'
         with:
           fetch-depth: 0
+          # If the workflow wasn't triggered by dispatch, this will be empty and use defaults.
+          ref: ${{ needs.dispatch-handler.outputs.head }}
```

Comment:
Will it properly reference the head in both WooCommerce and fork repositories?

Reply:
From my testing it looks like it does. Did you find it doesn't?

Reply:
Not yet, did first review sweep to understand changes and manual testing TBD yet.

Reply:
> Will it properly reference the head in both WooCommerce and fork repositories?

In PRs, `head` is the target branch in the same repository.